packages/frontend/src/services/indexer.js

```diff
@@ -1,17 +1,33 @@
 import { INDEXER_SERVICE_URL } from '../config';
 import sendJson from '../tmp_fetch_send_json';
+import { buildUrlOptParam } from '../utils/url';
+import IndexerCache from './lib/indexerCache';
+
+const cache = new IndexerCache();
+cache.open();
```

Comment:
Is there any need to check if cache is opened before making requests?
Is it possible that some request will throw if called immediately after cache.open ?